mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -166,9 +167,9 @@ public void onFailure(Call call, int statusCode, Headers headers, byte[] respons
 
     // Loopj
 //    private static MyAsyncHttpClient mAsyncHttpClient;
-    private static MyAsyncHttpClient mAsyncHttpClient;
+    public static MyAsyncHttpClient mAsyncHttpClient;
     // Base URL of current openHAB connection
-    private String openHABBaseUrl = "http://demo.openhab.org:8080/";
+    public static String openHABBaseUrl = "http://demo.openhab.org:8080/";
```

Comment:
Is making everything static here really a good idea? If it was me to decide, mAsyncHttpClient shouldn't be static to begin with, let alone public static. I'd suggest giving the service its own HTTP client and OpenHABTracker instance (like OpenHABVoiceService already has), and making CustomBroadcastReceiver a private non-static member class of that service so it has access to the client.